tests/08-native-runs/14-smallprng.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
```

Comment:
I think `#!/bin/sh -e` should be sufficient for this (bash  takes a long time to start compared to /bin/sh on most systems).

Reply:
:+1: